doc/administrator/galaxy/pipelines/mentalist/index.md

```diff
@@ -0,0 +1,84 @@
+
+## Step 2: Install Galaxy Tools
+
+Please install all the `mentalist` Galaxy tool by logging into Galaxy, navigating to **Admin > Search Tool Shed**, searching for `mentalist` and installing the appropriate **Toolshed Installable Revision**.
+
+The install progress can be checked by monitoring the Galaxy log files `galaxy/*.log`.  On completion you should see a message of `Installed` next to the tool when going to **Admin > Manage installed tool shed repositories**.
+
+## Step 3: Installing MentaLiST kmer Databases
+
+MentaLiST requires an organism-specific kmer database to search against. Log in to Galaxy and navigate to **Admin > Local data**. Under **Run Data Manager Tools**, select **MentaLiST Download from pubMLST**
+
+![mentalist-data-managers][]
+
+Choose your kmer size. The default kmer size of 31 should work well for most applications. From the drop-down menu, select 'Salmonella enterica'.
+
+![mentalist-download-pubmlst][]
+
+A green result box will appear in the galaxy history once the database is complete:
+
+![mentalist-download-pubmlst-result][]
+
+To confirm that the database is installed, select **mentalist_databases** from the list of available Tool Data Tables:
+
+![mentalist-data-table-list][]
+
+Your new MentaLiST database will be listed in the table of available MentaLiST databases. If it doesn't appear, click the refresh button at the top of the table.
+
+![mentalist-database-available][]
+
+## Step 4: Testing Pipeline
+
+A Galaxy workflow and some test data has been included with this documentation to verify that all tools are installed correctly.  To test this pipeline, please proceed through the following steps.
+
+1. Upload the [MentaLiST Galaxy Workflow][] by going to **Workflow > Upload or import workflow**.
+2. Upload the sequence reads by going to **Analyze Data** and then clicking on the **upload files from disk** icon ![upload-icon][].  Select the [test/reads][] files.  Make sure to change the **Type** of each file from **Auto-detect** to **fastqsanger**.  When uploaded you should see the following in your history.
+
+    ![upload-history][]
+
+3. Construct a dataset collection of the paired-end reads by clicking the **Operations on multiple datasets** icon ![datasets-icon][].  Please check off the two **.fastq** files and then go to **For all selected... > Build List of dataset pairs**.  You should see a screen that looks as follows.
+
+    ![dataset-pair-screen][]
+
+4. This should have properly paired your data and named the sample **AE014613-699860**.  Enter the name of this paired dataset collection at the bottom and click **Create list**.
+5. Run the uploaded workflow by clicking on **Workflow**, clicking on the name of the workflow **SISTR Analyze Reads v0.1 (imported from uploaded file)** and clicking **Run**.  This should auto fill in the dataset collection.  At the very bottom of the screen click **Run workflow**.
```

Comment:
This still says **SISTR** and the `workflow-success` screenshot does not show up.  Since this is just showing how to test if the tool runs (turns green) or fails (turns red) there's not too much else to change here beyond those and maybe mention that running the workflow should auto-fill the database (the data is Salmonella, but it will run for our purposes on any database).

Reply:
Thanks.